client/src/components/AuditAdmin/Progress/Progress.test.tsx

```diff
@@ -177,7 +177,7 @@ describe('Progress screen', () => {
       const footers = within(rows[5]).getAllByRole('cell')
       expect(footers[2]).toHaveTextContent('6,351')
       expect(footers[3]).toHaveTextContent('4,254')
-      expect(footers[4]).toHaveTextContent('30')
```

Comment:
I think the reason I didn't do this originally is because it made the total confusing. I thought the total should reflect the magnitude of the discrepancies in aggregate.

Reply:
Yeah the total is particularly tricky. I see arguments in both directions re what it should be. Fwiw @ginvdr requested this change and noted that she thought the total prior to this change was incorrect:

<table>
<img width="450" alt="slack" src="https://github.com/votingworks/arlo/assets/12616928/0604a43b-0ae4-482f-9679-a7dd96d8e9c4">
<table>

There's an argument to be made for not including a total for the `Difference From Expected Ballots` column at all

Reply:
Cool, went ahead and removed: https://github.com/votingworks/arlo/pull/1876/commits/1ea15dd57bb1b8adf837da549ea04cf9d8228340

And updated the PR description accordingly